gapic-libraries-bom/pom.xml

```diff
@@ -588,6 +588,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>com.google.maps</groupId>
+        <artifactId>google-maps-routes-bom</artifactId>
+        <version>0.0.1-SNAPSHOT</version><!-- {x-version-update:google-maps-routes:current} -->
```

Comment:
We need to exclude it from here by updating the script that re-generates the bom.